ledger/src/snapshot_utils.rs

```diff
@@ -192,7 +242,7 @@ pub fn archive_snapshot_package(snapshot_package: &AccountsPackage) -> Result<()
     // Write version file
     {
         let mut f = std::fs::File::create(staging_version_file)?;
-        f.write_all(&SNAPSHOT_VERSION.to_string().into_bytes())?;
+        f.write_all(DEFAULT_SNAPSHOT_VERSION.as_str().as_bytes())?;
```

Comment:
nits: well, the snapshot default version handling is duplicated by `DEFAULT_SNAPSHOT_VERSION` and `DefaultSerdeContext`. How about replacing `DEFAULT_SNAPSHOT_VERSION` with `DefaultSerdeContext::VERSION` or similar?

Reply:
SerdeContext and releated are no longer exposed to snapshot_utils, just SerdeStyle::OLDER and SerdeStyle::NEWER.